scripts/v.dissolve/v.dissolve.py

```diff
@@ -95,8 +398,17 @@ def main():
 
         if coltype["type"] not in ("INTEGER", "SMALLINT", "CHARACTER", "TEXT"):
             grass.fatal(_("Key column must be of type integer or string"))
+        column_is_str = bool(coltype["type"] in ("CHARACTER", "TEXT"))
+        if columns_to_aggregate and not column_is_str:
+            grass.fatal(
+                _(
+                    "Key column type must be string (text) "
+                    "for aggregation method to work, not '{column_type}'"
```

Comment:
Maybe, I'm missing something here. Why must the key column be string?

Reply:
r.reclass does not create the attribute table for ints. No idea why, but I didn't give it much thought. Here is what the manual says:

> No table is created for the reclassed map if the column option is used and the column type is integer (as the result could contain ambiguities).